src/makeInfinityAdder.js

```diff
@@ -4,7 +4,29 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  let i = 0;
```

Comment:
The variable 'i' is unnecessary for the logic of the function. You can remove it and simplify the condition checks.